features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql

```diff
@@ -1650,3 +1655,7 @@ CREATE INDEX IDX_POLICY_SET_REF ON IDN_XACML_POLICY_SET_REFERENCE (POLICY_ID, VE
 CREATE INDEX IDX_SUBSCRIBER_PROPERTY ON IDN_XACML_SUBSCRIBER_PROPERTY (SUBSCRIBER_ID, TENANT_ID);
 CREATE INDEX IDX_XACML_SUBSCRIBER_STATUS ON IDN_XACML_SUBSCRIBER_STATUS (SUBSCRIBER_ID, TENANT_ID);
 CREATE INDEX IDX_XACML_POLICY_STATUS ON IDN_XACML_POLICY_STATUS (POLICY_ID, POLICY_VERSION, TENANT_ID);
+
+--SAML--
+CREATE INDEX IDX_SAML2_CONFIG ON IDN_SAML2_SERVICE_PROVIDER (ISSUER,TENANT_ID);
+CREATE INDEX IDX_SAML2_ATTRIBUTES ON IDN_SAML2_SP_PROPERTIES (SP_ID,PROPERTY_NAME );
```

Comment:
Add a new line

Reply:
Resolved in [Update DAO Implementation according to new schema](https://github.com/wso2/carbon-identity-framework/pull/6014/commits/6f1cf3692c3be5519d5b8f505508017c64c55eec)